StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/DocumentationRules/SA1642SA1643CodeFixProvider.cs

```diff
@@ -99,29 +99,29 @@ private static Task<Document> GetTransformedDocumentAsync(Document document, Syn
             ImmutableArray<string> standardText;
             if (declarationSyntax is ConstructorDeclarationSyntax)
             {
-                var typeKindText = resourceManager.GetString(isStruct ? "TypeTextStruct" : "TypeTextClass", culture);
+                var typeKindText = resourceManager.GetString(isStruct ? nameof(DocumentationResources.TypeTextStruct) : nameof(DocumentationResources.TypeTextClass), culture);
```

Comment:
:memo: While I would love to just use the generated properties, this obviously isn't possible in a concurrent execution environment. I used `nameof` instead of string literals to get some of the benefits.